src/command.ts

```diff
@@ -241,7 +251,11 @@ export class Command<C extends Context = Context> implements MiddlewareObj<C> {
                 if (username && username !== ctx.me.username) continue;
                 if (
                     commandNames.some((name) =>
-                        matchesPattern(command.replace(prefix, ""), name)
+                        matchesPattern(
+                            command.replace(prefix, ""),
```

Comment:
Was this ever correct? If I use `e` as my prefix, won't that strip the commands of all `e`s? Feels like you wanted to do `command.substring(prefix.length)` or something?

Reply:
Oh damn, I somehow banned `replace` from my active memory because of that. I guess then it works